src/bench/ecdh.cpp

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) 2021 The Bitcoin Core developers
```

Comment:
nit: if you rebase this at some point, might as well update this

Reply:
Done.